docs/changelog/113286.yaml

```diff
@@ -0,0 +1,11 @@
+pr: 113286
+summary: Deprecate legacy params from range query
+area: Search
+type: deprecation
+issues: []
+deprecation:
+  title: Deprecate legacy params from range query
+  area: Search
+  details: Please describe the details of this change for the release notes. You can
+    use asciidoc.
+  impact: Please describe the impact of this change to users
```

Comment:
Notes should basically say the same thing as the original PR, indicating the parameters that are deprecated and that folks should use gt, lt, lte, gte, instead.

Reply:
@benwtrent Thanks, addressed in 6948107ec54cc003cccd3130e8d0d23bf660937b